curriculum/challenges/english/03-front-end-libraries/bootstrap/create-a-bootstrap-button.english.md

```diff
@@ -106,7 +106,70 @@ tests:
 ## Solution
 <section id='solution'>
 
-```js
-// solution required
-```
+<html>
```

Comment:
Your solution must be inside:
\`\`\`html
your solution
\`\`\`

Reply:
I took care of it for the user.  Why does it not display as code in the Preview on Github?